### DISCUSSION TITLE
Environment variable and install Datashare [Windows 10]

### DISCUSSION THREAD

**Discussion by Rainer-Maria** [ICIJ/datashare#1121] [Q&A]
Hi. 
Installing Datashare 11.4.0 on Windows 10 need probably something more to do.

I've this message : 'where' and '-cp' are unknown.

'where' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
'-cp' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.

I add a path variable in my user environment
Name : Datashare                      value : C:\Program Files\Datashare
But it doesn't work

Thanks a lot, all of you, for any help and advices
Rainer

**Comment by pirhoo** [comment]
Hi @Rainer-Maria,

This might be due to Java missing in your system. You can install it using this link:

https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.9.1+1/OpenJDK11U-jre_x64_windows_hotspot_11.0.9.1_1.msi

We will investigate the issue to ensure Datashare's installer check this correctly.

Best,

Pierre
